README.md

```diff
@@ -1,2 +1,87 @@
-# BuyingCatalogueDocumentService
-A service for retrieving files from Azure blob storage
+# BuyingCatalogueDocumentService - Service architecture for the NHS Digital Buying Catalogue Document Service .Net Core application.
+
+## IMPORTANT NOTES!
+**You can use either the latest version of Visual Studio or .NET CLI for Windows, Mac and Linux**.
+
+### Architecture overview
+This application uses <b>.NET core</b> to provide an API capable of running on Linux or Windows.
+It interfaces with Azure Blob Storage, via the .Net Azure Storage SDK.
+
+### Overview of the application code
+This repo consists of one service using <b>.NET Core</b> and <b>Docker</b>.
+
+It contains two main endpoints:
+- api/v1/Solutions/{solutionId}/documents
```

Comment:
Perhaps add a new line before the first list item so that the markdown? The markdown is closer to its rendered version then and it will also ensure compatibility with the majority of parsers (I believe some won't parse lists that don't have a space before and after).